api/services/ocean/rightsizing/schemas/oceanRightsizingRuleRecommendationApplicationHPA.yaml

```diff
@@ -0,0 +1,9 @@
+type: object
+title: Ocean Rightsizing Rule Recommendation Application For HPA
+description: >
+  Determines by the rule if recommendation application is allowed for workloads with HPA definition.
+properties:
+  allowHPARecommendation:
```

Comment:
The correct field name in the spotinst-api code is "allowHPARecommendations" with 's' in the end.

Reply:
fixed